static/js/BookPage.jsx

```diff
@@ -813,6 +822,22 @@ ArrayMapNode.propTypes = {
 class DictionaryNode extends Component {
   render() {
     if (this.props.schema.headwordMap) {
+      const dictionarySearch =
+              <DictionarySearch
+              lexiconName={this.props.lexiconName}
+              title={this.props.title}
+              showBaseText={this.props.showBaseText}
+              contextSelector=".bookPage"
+              currVersions={this.props.currVersions}/>;
+      const header = this.props.schema.title ? (
```

Comment:
The only thing that changes if there is a schema title is the text passed to `ContentText`. Best to calculate the text beforehand and then you don't need an if statement to create the header.